### ISSUE TITLE
Screen Reader doesn't read About Dialogue Box

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#19928]
### Issue type

UI bug

### Bug description

There is no current way for a screen reader to access the info in this dialogue box.  It would be ideal if it not only read, but could be arrowed around to read character by character and selected/copied.

### Steps to reproduce

1. From the Help menu choose About.
The About dialog appears.

Expected behaviour: The text in the About dialog should be read

Actual behaviour: the title of the About dialog is read.

Suggested fix: make the text in the About dialog a read-only edit box. Add OK(or Close) button. Make both tabbable objects.

### MuseScore Version

4.1

### Regression

I don't know

### Operating system

Windows 10

**Comment by jmwmusic** [comment]
There is a similar issue reading text in some other dialogue boxes.  Noteably the new update available window.  The actionable buttons such as update can be tabbed/arrowed to, but if for any reason the speech is interrupted that reads the main window of what the update includes it can't be restarted, it can't be arrowed over to hear bit by bit and there is no way to get to the links to activate them with keyboard.
